src/extension/commandPaletteHandler.ts

```diff
@@ -37,7 +40,29 @@ export class CommandPaletteHandler {
      * Executes the 'react-native run-android' command
      */
     public runAndroid(): Q.Promise<void> {
-        return this.executeCommandInContext("runAndroid", () => this.executeReactNativeRunCommand("run-android"));
+        return this.executeCommandInContext("runAndroid", () => this.executeReactNativeRunCommand("run-android"))
```

Comment:
Please add a comment stating that `react-native run-android` installs on all devices, but fails to launch on them despite reporting success, and thus we manually launch if there is more than one device attached. Once/if they fix that, we'll need to revisit this I suppose.